### ISSUE TITLE
problem INSTALLING  ionic(in cli)

### ISSUE THREAD

**Issue by edgbk** [driftyco/ionic-cli#193]
HI , I FOLLOWED INSTRUCTIONS AT http://ionicframework.com/docs/guide/installation.html
BUT I GET THIS RESULTS ;CAN ANY ONE HELP
Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\Sena>npm install -g ionic
C:\Users\Sena\AppData\Roaming\npm\ionic -> C:\Users\Sena\AppData\Roaming\npm\nod
e_modules\ionic\bin\ionic
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\npm requires npmconf@'~1.1.8' but will lo
ad
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\npmconf,
npm WARN unmet dependency which is version 0.1.16
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\npm requires request@'~2.42.0' but will l
oad
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\request,
npm WARN unmet dependency which is version 2.47.0
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\npm requires semver@'~2.3.0' but will loa
d
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\semver,
npm WARN unmet dependency which is version 2.0.11
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\cordova-js\node_modules\browserify requir
es through2@'^1.0.0' but will load
npm WARN unmet dependency C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
\node_modules\cordova-lib\node_modules\through2,
npm WARN unmet dependency which is version 0.6.3
ionic@1.3.2 C:\Users\Sena\AppData\Roaming\npm\node_modules\ionic
├── proxy-middleware@0.7.0
├── connect-livereload@0.4.0
├── progress@1.1.7
├── colors@0.6.2
├── open@0.0.5
├── ncp@0.4.2
├── underscore@1.7.0
├── crc@3.2.1
├── q@1.0.1
├── cross-spawn@0.2.3 (lru-cache@2.5.0)
├── shelljs@0.2.6
├── event-stream@3.0.20 (stream-combiner@0.0.4, duplexer@0.1.1, from@0.1.3, paus
e-stream@0.0.11, map-stream@0.0.5, split@0.2.10, through@2.3.6)
├── finalhandler@0.2.0 (escape-html@1.0.1, debug@2.0.0)
├── optimist@0.6.0 (wordwrap@0.0.2, minimist@0.0.10)
├── connect@3.1.1 (utils-merge@1.0.0, parseurl@1.3.0, finalhandler@0.1.0, debug@
1.0.4)
├── form-data@0.1.4 (mime@1.2.11, async@0.9.0, combined-stream@0.0.7)
├── serve-static@1.7.1 (utils-merge@1.0.0, escape-html@1.0.1, parseurl@1.3.0, se
nd@0.10.1)
├── opbeat@1.1.0 (json-stringify-safe@5.0.0, console-log-level@1.0.0, stackman@0
.2.2, after-all@2.0.1)
├── tiny-lr-fork@0.0.5 (debug@0.7.4, faye-websocket@0.4.4, qs@0.5.6, noptify@0.0
.3)
├── unzip@0.1.9 (setimmediate@1.0.2, pullstream@0.4.1, match-stream@0.0.2, reada
ble-stream@1.0.33, fstream@0.1.31, binary@0.3.0)
├── request@2.51.0 (caseless@0.8.0, json-stringify-safe@5.0.0, forever-agent@0.5
.2, aws-sign2@0.5.0, stringstream@0.0.4, oauth-sign@0.5.0, tunnel-agent@0.4.0, n
ode-uuid@1.4.2, qs@2.3.3, mime-types@1.0.2, combined-stream@0.0.7, http-signatur
e@0.10.1, tough-cookie@0.12.1, bl@0.9.3, form-data@0.2.0, hawk@1.1.1)
├── prompt@0.2.12 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.5, utile@0.2.1, wi
nston@0.6.2)
├── vinyl-fs@0.3.7 (graceful-fs@3.0.5, strip-bom@1.0.0, vinyl@0.4.6, mkdirp@0.5.
0, through2@0.6.3, glob-stream@3.1.18, lodash@2.4.1, glob-watcher@0.0.6)
├── archiver@0.5.1 (readable-stream@1.1.13, zip-stream@0.1.4, lazystream@0.1.0,
lodash@2.4.1, file-utils@0.1.5)
├── gulp@3.8.8 (pretty-hrtime@0.2.2, interpret@0.3.10, deprecated@0.0.1, archy@0
.0.2, minimist@1.1.0, tildify@1.0.0, semver@3.0.1, chalk@0.5.1, orchestrator@0.3
.7, gulp-util@3.0.2, liftoff@0.12.1)
├── xml2js@0.4.4 (sax@0.6.1, xmlbuilder@2.4.5)
└── npm@2.1.3

C:\Users\Sena>ionic start todo blank
'ionic' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Sena>

**Comment by jbavari** [comment]
@edgbk Do you have cordova installed? `npm install -g cordova`

**Comment by edgbk** [comment]
hmm did install cordova before but since u ask i'll try again(i appreciate your help)
hmm dont know what to make of this problem!

Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\Sena>npm install -g cordova
C:\Users\Sena\AppData\Roaming\npm\cordova -> C:\Users\Sena\AppData\Roaming\npm\n
ode_modules\cordova\bin\cordova
cordova@4.2.0 C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
├── underscore@1.7.0
├── q@1.0.1
├── nopt@3.0.1 (abbrev@1.0.5)
└── cordova-lib@4.2.0 (valid-identifier@0.0.1, osenv@0.1.0, properties-parser@0.
2.3, unorm@1.3.3, mime@1.2.11, bplist-parser@0.0.6, semver@2.0.11, shelljs@0.3.0
, dep-graph@1.1.0, rc@0.5.2, through2@0.6.3, xcode@0.6.7, npmconf@0.1.16, d8@0.4
.4, elementtree@0.1.5, glob@4.0.6, tar@1.0.2, request@2.47.0, plist@1.1.0, npm@1
.4.28, cordova-js@3.7.3)

C:\Users\Sena>npm install -g ionic
C:\Users\Sena\AppData\Roaming\npm\ionic -> C:\Users\Sena\AppData\Roaming\npm\nod
e_modules\ionic\bin\ionic
ionic@1.3.2 C:\Users\Sena\AppData\Roaming\npm\node_modules\ionic
├── proxy-middleware@0.7.0
├── connect-livereload@0.4.0
├── progress@1.1.7
├── colors@0.6.2
├── open@0.0.5
├── ncp@0.4.2
├── underscore@1.7.0
├── crc@3.2.1
├── q@1.0.1
├── shelljs@0.2.6
├── cross-spawn@0.2.3 (lru-cache@2.5.0)
├── event-stream@3.0.20 (stream-combiner@0.0.4, duplexer@0.1.1, from@0.1.3, map-
stream@0.0.5, pause-stream@0.0.11, split@0.2.10, through@2.3.6)
├── finalhandler@0.2.0 (escape-html@1.0.1, debug@2.0.0)
├── optimist@0.6.0 (wordwrap@0.0.2, minimist@0.0.10)
├── connect@3.1.1 (utils-merge@1.0.0, parseurl@1.3.0, finalhandler@0.1.0, debug@
1.0.4)
├── form-data@0.1.4 (async@0.9.0, mime@1.2.11, combined-stream@0.0.7)
├── opbeat@1.1.0 (json-stringify-safe@5.0.0, console-log-level@1.0.0, stackman@0
.2.2, after-all@2.0.1)
├── serve-static@1.7.1 (utils-merge@1.0.0, parseurl@1.3.0, escape-html@1.0.1, se
nd@0.10.1)
├── tiny-lr-fork@0.0.5 (debug@0.7.4, faye-websocket@0.4.4, qs@0.5.6, noptify@0.0
.3)
├── unzip@0.1.9 (setimmediate@1.0.2, pullstream@0.4.1, match-stream@0.0.2, reada
ble-stream@1.0.33, binary@0.3.0, fstream@0.1.31)
├── request@2.51.0 (caseless@0.8.0, json-stringify-safe@5.0.0, forever-agent@0.5
.2, aws-sign2@0.5.0, stringstream@0.0.4, oauth-sign@0.5.0, tunnel-agent@0.4.0, n
ode-uuid@1.4.2, qs@2.3.3, mime-types@1.0.2, combined-stream@0.0.7, http-signatur
e@0.10.1, tough-cookie@0.12.1, bl@0.9.3, form-data@0.2.0, hawk@1.1.1)
├── prompt@0.2.12 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.5, utile@0.2.1, wi
nston@0.6.2)
├── vinyl-fs@0.3.7 (graceful-fs@3.0.5, lodash@2.4.1, strip-bom@1.0.0, vinyl@0.4.
6, mkdirp@0.5.0, through2@0.6.3, glob-stream@3.1.18, glob-watcher@0.0.6)
├── archiver@0.5.1 (lodash@2.4.1, readable-stream@1.1.13, lazystream@0.1.0, zip-
stream@0.1.4, file-utils@0.1.5)
├── gulp@3.8.8 (interpret@0.3.10, pretty-hrtime@0.2.2, deprecated@0.0.1, archy@0
.0.2, minimist@1.1.0, semver@3.0.1, tildify@1.0.0, chalk@0.5.1, orchestrator@0.3
.7, gulp-util@3.0.2, liftoff@0.12.1)
├── xml2js@0.4.4 (sax@0.6.1, xmlbuilder@2.4.5)
└── npm@2.1.3

C:\Users\Sena>ionic start todo blank
'ionic' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Sena>install -g ionic
'install' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Sena>npm install -g ionic
C:\Users\Sena\AppData\Roaming\npm\ionic -> C:\Users\Sena\AppData\Roaming\npm\nod
e_modules\ionic\bin\ionic
ionic@1.3.2 C:\Users\Sena\AppData\Roaming\npm\node_modules\ionic
├── proxy-middleware@0.7.0
├── connect-livereload@0.4.0
├── progress@1.1.7
├── open@0.0.5
├── colors@0.6.2
├── ncp@0.4.2
├── underscore@1.7.0
├── crc@3.2.1
├── q@1.0.1
├── shelljs@0.2.6
├── cross-spawn@0.2.3 (lru-cache@2.5.0)
├── event-stream@3.0.20 (stream-combiner@0.0.4, duplexer@0.1.1, from@0.1.3, paus
e-stream@0.0.11, map-stream@0.0.5, split@0.2.10, through@2.3.6)
├── optimist@0.6.0 (wordwrap@0.0.2, minimist@0.0.10)
├── finalhandler@0.2.0 (escape-html@1.0.1, debug@2.0.0)
├── connect@3.1.1 (utils-merge@1.0.0, parseurl@1.3.0, finalhandler@0.1.0, debug@
1.0.4)
├── form-data@0.1.4 (mime@1.2.11, async@0.9.0, combined-stream@0.0.7)
├── opbeat@1.1.0 (json-stringify-safe@5.0.0, console-log-level@1.0.0, stackman@0
.2.2, after-all@2.0.1)
├── serve-static@1.7.1 (escape-html@1.0.1, utils-merge@1.0.0, parseurl@1.3.0, se
nd@0.10.1)
├── tiny-lr-fork@0.0.5 (debug@0.7.4, faye-websocket@0.4.4, qs@0.5.6, noptify@0.0
.3)
├── unzip@0.1.9 (setimmediate@1.0.2, match-stream@0.0.2, readable-stream@1.0.33,
 pullstream@0.4.1, fstream@0.1.31, binary@0.3.0)
├── request@2.51.0 (caseless@0.8.0, json-stringify-safe@5.0.0, forever-agent@0.5
.2, aws-sign2@0.5.0, oauth-sign@0.5.0, tunnel-agent@0.4.0, stringstream@0.0.4, q
s@2.3.3, mime-types@1.0.2, node-uuid@1.4.2, combined-stream@0.0.7, tough-cookie@
0.12.1, bl@0.9.3, http-signature@0.10.1, form-data@0.2.0, hawk@1.1.1)
├── prompt@0.2.12 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.5, utile@0.2.1, wi
nston@0.6.2)
├── vinyl-fs@0.3.7 (graceful-fs@3.0.5, strip-bom@1.0.0, vinyl@0.4.6, mkdirp@0.5.
0, lodash@2.4.1, through2@0.6.3, glob-stream@3.1.18, glob-watcher@0.0.6)
├── archiver@0.5.1 (readable-stream@1.1.13, lazystream@0.1.0, zip-stream@0.1.4,
lodash@2.4.1, file-utils@0.1.5)
├── gulp@3.8.8 (pretty-hrtime@0.2.2, interpret@0.3.10, deprecated@0.0.1, archy@0
.0.2, minimist@1.1.0, tildify@1.0.0, semver@3.0.1, chalk@0.5.1, orchestrator@0.3
.7, gulp-util@3.0.2, liftoff@0.12.1)
├── xml2js@0.4.4 (sax@0.6.1, xmlbuilder@2.4.5)
└── npm@2.1.3

C:\Users\Sena>ionic start todo blank
'ionic' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Sena>

**Comment by jbavari** [comment]
Its looking like its not letting you hit the ionic command for some reason. 

Try [this solution](http://stackoverflow.com/questions/27454810/ionic-is-not-recognized-as-an-internal-or-external-command)

**Comment by edgbk** [comment]
hmm just don't get it !

Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\Sena>npm cache clean

C:\Users\Sena>npm install -g cordova
C:\Users\Sena\AppData\Roaming\npm\cordova -> C:\Users\Sena\AppData\Roaming\npm\n
ode_modules\cordova\bin\cordova
cordova@4.2.0 C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
├── underscore@1.7.0
├── q@1.0.1
├── nopt@3.0.1 (abbrev@1.0.5)
└── cordova-lib@4.2.0 (valid-identifier@0.0.1, osenv@0.1.0, properties-parser@0.
2.3, bplist-parser@0.0.6, mime@1.2.11, unorm@1.3.3, semver@2.0.11, shelljs@0.3.0
, rc@0.5.2, dep-graph@1.1.0, elementtree@0.1.5, xcode@0.6.7, npm@1.4.28, through
2@0.6.3, npmconf@0.1.16, glob@4.0.6, tar@1.0.2, d8@0.4.4, request@2.47.0, plist@
1.1.0, cordova-js@3.7.3)

C:\Users\Sena>npm install -g ionic
C:\Users\Sena\AppData\Roaming\npm\ionic -> C:\Users\Sena\AppData\Roaming\npm\nod
e_modules\ionic\bin\ionic
ionic@1.3.2 C:\Users\Sena\AppData\Roaming\npm\node_modules\ionic
├── proxy-middleware@0.7.0
├── connect-livereload@0.4.0
├── progress@1.1.7
├── colors@0.6.2
├── open@0.0.5
├── ncp@0.4.2
├── underscore@1.7.0
├── crc@3.2.1
├── q@1.0.1
├── shelljs@0.2.6
├── cross-spawn@0.2.3 (lru-cache@2.5.0)
├── event-stream@3.0.20 (duplexer@0.1.1, stream-combiner@0.0.4, from@0.1.3, paus
e-stream@0.0.11, map-stream@0.0.5, split@0.2.10, through@2.3.6)
├── optimist@0.6.0 (wordwrap@0.0.2, minimist@0.0.10)
├── finalhandler@0.2.0 (escape-html@1.0.1, debug@2.0.0)
├── form-data@0.1.4 (mime@1.2.11, async@0.9.0, combined-stream@0.0.7)
├── connect@3.1.1 (utils-merge@1.0.0, parseurl@1.3.0, finalhandler@0.1.0, debug@
1.0.4)
├── opbeat@1.1.0 (json-stringify-safe@5.0.0, console-log-level@1.0.0, stackman@0
.2.2, after-all@2.0.1)
├── serve-static@1.7.1 (escape-html@1.0.1, utils-merge@1.0.0, parseurl@1.3.0, se
nd@0.10.1)
├── tiny-lr-fork@0.0.5 (debug@0.7.4, faye-websocket@0.4.4, qs@0.5.6, noptify@0.0
.3)
├── unzip@0.1.9 (setimmediate@1.0.2, readable-stream@1.0.33, match-stream@0.0.2,
 pullstream@0.4.1, binary@0.3.0, fstream@0.1.31)
├── prompt@0.2.12 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.5, utile@0.2.1, wi
nston@0.6.2)
├── request@2.51.0 (caseless@0.8.0, aws-sign2@0.5.0, forever-agent@0.5.2, oauth-
sign@0.5.0, tunnel-agent@0.4.0, json-stringify-safe@5.0.0, stringstream@0.0.4, q
s@2.3.3, mime-types@1.0.2, node-uuid@1.4.2, tough-cookie@0.12.1, combined-stream
@0.0.7, http-signature@0.10.1, hawk@1.1.1, form-data@0.2.0, bl@0.9.3)
├── vinyl-fs@0.3.7 (graceful-fs@3.0.5, lodash@2.4.1, strip-bom@1.0.0, vinyl@0.4.
6, mkdirp@0.5.0, through2@0.6.3, glob-stream@3.1.18, glob-watcher@0.0.6)
├── archiver@0.5.1 (lodash@2.4.1, readable-stream@1.1.13, lazystream@0.1.0, zip-
stream@0.1.4, file-utils@0.1.5)
├── gulp@3.8.8 (pretty-hrtime@0.2.2, interpret@0.3.10, deprecated@0.0.1, archy@0
.0.2, semver@3.0.1, minimist@1.1.0, tildify@1.0.0, chalk@0.5.1, orchestrator@0.3
.7, gulp-util@3.0.2, liftoff@0.12.1)
├── npm@2.1.3
└── xml2js@0.4.4 (sax@0.6.1, xmlbuilder@2.4.5)

C:\Users\Sena>ionic start todo blank
'ionic' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Sena>ionic start todo blank
'ionic' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Sena>

**Comment by mattezell** [comment]
Yeah, this is getting me too I believe... 

I've been using Cordova with Ionic for about 2 months now... I recently upgraded the Ionic CLI after being prompted and now I no longer have access to Ionic or Cordova (not saying the upgrade is the cause, just that it's the last thing that I recall doing to this system other than developing using it)...  

I've since attempted to resolve my issue via a npm cache clear and a reinstall of Cordova and Ionic, but I am still having path issues.. 

NPM executes without issue or error, but Ionic nor Cordova appear to be in the path anymore (which, if I recall, both are added when installing via 'npm install -g').... 

Fun times...

**Comment by mattezell** [comment]
Update in case it helps someone else...

I am not sure what's caused this - I've also found a recent thread in the Ionic forums that seems to be related... Everything was fine a few days ago and then seemingly something broke... Windows Update?  Upgrading Node? NPM? Ionic?  No idea...

To resolve this, I had to edit my path as advised here: https://github.com/npm/npm/wiki/Troubleshooting#upgrading-on-windows

Basically, I had to add the Windows user path for NPM modules to my path before the installer inserted path to NodeJS... This raises the question (though not an Ionic question, but a NodeJS question) - Why does the NodeJS installer only add the '\program files (x86)\' path when NPM clearly installs modules to the '...\<user>\AppData\Roaming\npm\...' path?

**Comment by jbavari** [comment]
@mattezell - thanks for sharing that. 

Did that take care of it for you? 

@edgbk - you might want to try those suggestions Matt made.

**Comment by mattezell** [comment]
Yeah, adding the '...\AppData\Roaming\...' npm path to my PATH variable before the NodeJS-installer-added path for '...\Program Files (x86)\nodejs\' resolved the '..is not a recognized..' issues I was encountered when trying to run Cordova or Ionic... 

I am still scratching my head as to how this happened, but the best that I can figure is that something modified my path, though I have no idea what... I kid you not - I was developing on an Ionic project until about 1am this morning without issue.  

Chain of events was that a corp update was pushed that required reboot - run of the mill Windows update, so far as I know...  After reboot, a cmd window was auto-opened upon first login by Windows - I used this cmd window for development all day without issue (lots of "ionic run android -l -c -s" being executed without a problem)... Today I come in and decide to start debugging both of the apps I am writing (a 'phone' and a 'tablet' app using Ionic that communicate) and decide to fire up another cmd window so I can actively debug both... In the new cmd window, 'ionic' results in the '..is not recognized..' issue first hitting my radar... I then close both cmd windows to go ahead and bite the bullet to research this issue - at which time I realize that all newly opened command windows no longer have access to the Ionic path...

The more that I think about it, I am guessing (just a guess) that whatever mechanism re-opens open applications upon first login after a update-forced reboot perhaps also provides whatever PATH was passed in when it was initially opened, which perhaps somehow contained the \AppData\ path which allowed it to resolve Ionic... Then manually opened CMD windows don't have this PATH passed to them as the actual system PATH had somehow been modified by the upgrade or some other mechanism...  

What I don't have any clue about is, if this guess is correct, why didn't just reinstalling NodeJS/NPM via the Windows MSI resolve the issue?  Has something changed with the installer in regards to the paths that it adds to the PATH?  I suppose it's possible that I added this \AppData\ user npm path to my PATH when I first installed this stack 3 months ago, but I definitely do not recall doing such a thing.. Additionally to this point, everything that I read in discussions on this say that the Windows installer should properly add the NPM modules path - but then that begs the question of why the information in the link I shared exists to begin with if this isn't a common problem...

**Comment by edgbk** [comment]
hurrayyyy!!!! (:
seems to be be responding at last.{thanks guys for all the help}

Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\Sena>npm cache clean

C:\Users\Sena>npm install -g cordova
C:\Users\Sena\AppData\Roaming\npm\cordova -> C:\Users\Sena\AppData\Roaming\npm\n
ode_modules\cordova\bin\cordova
cordova@4.2.0 C:\Users\Sena\AppData\Roaming\npm\node_modules\cordova
├── q@1.0.1
├── underscore@1.7.0
├── nopt@3.0.1 (abbrev@1.0.5)
└── cordova-lib@4.2.0 (valid-identifier@0.0.1, osenv@0.1.0, properties-parser@0.
2.3, bplist-parser@0.0.6, mime@1.2.11, unorm@1.3.3, semver@2.0.11, shelljs@0.3.0
, rc@0.5.2, dep-graph@1.1.0, xcode@0.6.7, elementtree@0.1.5, npmconf@0.1.16, thr
ough2@0.6.3, glob@4.0.6, tar@1.0.2, d8@0.4.4, request@2.47.0, plist@1.1.0, cordo
va-js@3.7.3, npm@1.4.28)

C:\Users\Sena>npm install -g ionic
C:\Users\Sena\AppData\Roaming\npm\ionic -> C:\Users\Sena\AppData\Roaming\npm\nod
e_modules\ionic\bin\ionic
ionic@1.3.2 C:\Users\Sena\AppData\Roaming\npm\node_modules\ionic
├── proxy-middleware@0.7.0
├── connect-livereload@0.4.0
├── progress@1.1.7
├── colors@0.6.2
├── open@0.0.5
├── ncp@0.4.2
├── underscore@1.7.0
├── crc@3.2.1
├── q@1.0.1
├── shelljs@0.2.6
├── cross-spawn@0.2.3 (lru-cache@2.5.0)
├── event-stream@3.0.20 (from@0.1.3, stream-combiner@0.0.4, duplexer@0.1.1, paus
e-stream@0.0.11, map-stream@0.0.5, split@0.2.10, through@2.3.6)
├── finalhandler@0.2.0 (escape-html@1.0.1, debug@2.0.0)
├── connect@3.1.1 (utils-merge@1.0.0, parseurl@1.3.0, debug@1.0.4, finalhandler@
0.1.0)
├── optimist@0.6.0 (wordwrap@0.0.2, minimist@0.0.10)
├── form-data@0.1.4 (mime@1.2.11, async@0.9.0, combined-stream@0.0.7)
├── opbeat@1.1.0 (json-stringify-safe@5.0.0, console-log-level@1.0.0, stackman@0
.2.2, after-all@2.0.1)
├── serve-static@1.7.1 (utils-merge@1.0.0, parseurl@1.3.0, escape-html@1.0.1, se
nd@0.10.1)
├── tiny-lr-fork@0.0.5 (debug@0.7.4, faye-websocket@0.4.4, qs@0.5.6, noptify@0.0
.3)
├── unzip@0.1.9 (setimmediate@1.0.2, readable-stream@1.0.33, pullstream@0.4.1, m
atch-stream@0.0.2, binary@0.3.0, fstream@0.1.31)
├── request@2.51.0 (caseless@0.8.0, json-stringify-safe@5.0.0, forever-agent@0.5
.2, aws-sign2@0.5.0, stringstream@0.0.4, oauth-sign@0.5.0, tunnel-agent@0.4.0, n
ode-uuid@1.4.2, qs@2.3.3, mime-types@1.0.2, combined-stream@0.0.7, tough-cookie@
0.12.1, http-signature@0.10.1, form-data@0.2.0, bl@0.9.3, hawk@1.1.1)
├── prompt@0.2.12 (revalidator@0.1.8, pkginfo@0.3.0, read@1.0.5, utile@0.2.1, wi
nston@0.6.2)
├── vinyl-fs@0.3.7 (graceful-fs@3.0.5, lodash@2.4.1, strip-bom@1.0.0, vinyl@0.4.
6, mkdirp@0.5.0, through2@0.6.3, glob-stream@3.1.18, glob-watcher@0.0.6)
├── archiver@0.5.1 (lodash@2.4.1, readable-stream@1.1.13, zip-stream@0.1.4, lazy
stream@0.1.0, file-utils@0.1.5)
├── gulp@3.8.8 (interpret@0.3.10, archy@0.0.2, deprecated@0.0.1, pretty-hrtime@0
.2.2, minimist@1.1.0, semver@3.0.1, tildify@1.0.0, chalk@0.5.1, orchestrator@0.3
.7, gulp-util@3.0.2, liftoff@0.12.1)
├── xml2js@0.4.4 (sax@0.6.1, xmlbuilder@2.4.5)
└── npm@2.1.3

C:\Users\Sena>ionic start todo blank
Creating Ionic app in folder C:\Users\Sena\todo based on blank project

Downloading: https://github.com/driftyco/ionic-app-base/archive/master.zip
[=============================]  100%  0.0s

Downloading: https://github.com/driftyco/ionic-starter-blank/archive/master.zip
[=============================]  100%  0.0s

Update config.xml
Initializing cordova project
Fetching plugin "org.apache.cordova.device" via plugin registry
Fetching plugin "org.apache.cordova.console" via plugin registry
Fetching plugin "com.ionic.keyboard" via plugin registry

Your Ionic project is ready to go! Some quick tips:

 * cd into your project: $ cd todo

 * Setup this project to use Sass: ionic setup sass

 * Develop in the browser with live reload: ionic serve

 * Add a platform (ios or Android): ionic platform add ios [android]
   Note: iOS development requires OS X currently
   See the Android Platform Guide for full Android installation instructions:
   https://cordova.apache.org/docs/en/edge/guide_platforms_android_index.md.html


 * Build your app: ionic build <PLATFORM>

 * Simulate your app: ionic emulate <PLATFORM>

 * Run your app on a device: ionic run <PLATFORM>

 * Package an app using Ionic package service: ionic package <MODE> <PLATFORM>

For more help use ionic --help or visit the Ionic docs: http://ionicframework.co
m/docs


C:\Users\Sena>

**Comment by codrinf** [comment]
I had the same situation:

I've used ionic without problems for couple of days, than made some changes on my PATH variable due the reinstall of Android sdk. I may deleted or changed some chars there, don't know :)
Anyway, here is my PATH now and Ionic is working:

*other vars*;C:\Program Files (x86)\Git\cmd;C:\Program Files (x86)\Git\bin;C:\Program Files (x86)\Android\android-sdk\tools\;C:\Program Files (x86)\Android\android-sdk\platform-tools\;C:\Users\Codrin\AppData\Roaming\npm;C:\Program Files\nodejs
 
Basically I added C:\Users\Codrin\AppData\Roaming\npm BEFORE C:\Program Files\nodejs
